app/assets/stylesheets/components/_event.sass

```diff
@@ -1,17 +1,55 @@
 $Event-color-background: Theme-color(background, event-card)
+$Event-color-label: Theme-color(text, label)
+$Event-color-primaryInfo: Theme-color(text, event-primary-info)
+$Event-color-secondaryInfo: Theme-color(text, event-secondary-info)
 $Event-color-title: Theme-color(text, title-event)
-$Event-height: 150px
-$Event-spacing-base: $Theme-spacing-base
-$Event-width: 280px
+$Event-fontSize-label: Theme-fontSize(label, font-size)
+$Event-fontSize-primaryInfo: Theme-fontSize(event-primary-info, font-size)
+$Event-fontSize-secondaryInfo: Theme-fontSize(event-secondary-info, font-size)
+$Event-fontSize-title: Theme-fontSize(event-title, font-size)
+$Event-fontWeight-label: Theme-fontSize(label, font-weight)
+$Event-fontWeight-primaryInfo: Theme-fontSize(event-primary-info, font-weight)
+$Event-fontWeight-secondaryInfo: Theme-fontSize(event-secondary-info, font-weight)
+$Event-fontWeight-title: Theme-fontSize(event-title, font-weight)
+$Event-lineHeight-label: Theme-fontSize(label, line-height)
+$Event-lineHeight-primaryInfo: Theme-fontSize(event-primary-info, line-height)
+$Event-lineHeight-secondaryInfo: Theme-fontSize(event-secondary-info, line-height)
+$Event-lineHeight-title: Theme-fontSize(event-title, line-height)
+$Event-spacing-base: 23px
+$Event-radius: 2px
+
```

Comment:
I thought in keep the height and width of the event together. But it can be together and without the empty line.